<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>901</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pandemie Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="graph" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>521</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="healthyLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>132</y>
      <width>521</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 158, 37);
</string>
    </property>
    <property name="text">
     <string>000 Gesund</string>
    </property>
   </widget>
   <widget class="QLabel" name="infectedLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>118</y>
      <width>521</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 0, 0);</string>
    </property>
    <property name="text">
     <string>000 Infiziert</string>
    </property>
   </widget>
   <widget class="QLabel" name="recoveredLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>104</y>
      <width>521</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(12, 0, 193);</string>
    </property>
    <property name="text">
     <string>000 Genesen</string>
    </property>
   </widget>
   <widget class="QLabel" name="deadLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>521</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(136, 136, 136);</string>
    </property>
    <property name="text">
     <string>000 Gestorben</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>630</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zeit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Anteil 
Menschen 
in %</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>720</y>
      <width>1231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="countInput">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>760</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>740</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gesamtzahl Menschen</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>730</y>
      <width>16</width>
      <height>81</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="socialDistancingCb">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>730</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Social Distancing</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>790</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 14pt &quot;Arial Black&quot;;
color: rgb(211, 211, 211);</string>
    </property>
    <property name="text">
     <string>Maßnahmen</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="socialDistancingInp">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>770</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>750</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Menschen</string>
    </property>
   </widget>
   <widget class="QComboBox" name="socialDistancingDd">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>770</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Anzahl</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>%</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>750</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Anzahl / %</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>730</y>
      <width>16</width>
      <height>81</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>760</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>730</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 14pt &quot;Arial Black&quot;;
color: rgb(211, 211, 211);</string>
    </property>
    <property name="text">
     <string>Start Simulation</string>
    </property>
   </widget>
   <widget class="QLabel" name="groupLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 14pt &quot;Arial Black&quot;;
color: rgb(211, 211, 211);</string>
    </property>
    <property name="text">
     <string>Gruppe mit Maßnahmen</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="view">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="infectedInput">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>810</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>790</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Anfänglich Infizierte</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1270</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
